src/objects.js

```diff
@@ -1,42 +1,70 @@
 const createPerson = (name, age) => {
-  // your code here
+  const person = {name:name, age:age};
```

Comment:
if your variable name matches the name you want to give to that property in an object, you can just pass it the variable, i.e. `{ name, age }`.
This will create an object with the property's key being the name of the variable and the properties value being the value of the variable.